re:Invent banner and misc get-started fixes (#4838)
SumoLogic/sumologic-documentation: docs/get-started/ai-machine-learning.md

</details>

> Our AI and security analytics capabilities allow security and development teams to align around a single source of truth and collect and act on data insights more quickly.

Reviewer: ```suggestion
Our AI and security analytics capabilities enable security and development teams to align around a unified source of truth, allowing for quicker collection and action on data insights.
```

Author: thank you!! good one!
